src-web/components/common/TemplateEditor/scss/template-editor.scss

```diff
@@ -25,29 +25,16 @@
 
   .creation-view-split-container {
     width: 100%;
-
-    div.SplitPane {
-      div.Pane1 {
-        position: unset !important;
-      }
-    }
-
-    .creation-view-controls-container {
-      margin-top: 30px;
-    }
-
     &.showEditor {
-      height: calc(100vh - 190px);
-
+      // height: calc(100vh - 190px);
```

Comment:
can these comments be removed?

Reply:
will remove in future clean up pr

Reply:
I updated the patternfly css to latest so in theory we should no longer need to override css. Can you test it again after the pr is merged? It it doesn’t work, we can add css back